tests/nist/test_worker.py

```diff
@@ -79,7 +79,7 @@ def run_test(test, scanner):
         sys.stderr.write("Unable to run the test! Check the catalog.xml.\n")
         return None
     print(command)
-    with open(test_id + ".stdout", "w") as stdout_file:
+    with open(os.path.join(output_dir, test_id + ".stdout"), "w") as stdout_file:
```

Comment:
When running `test_worker.py` by hand and specifying `output_dir`, `output_dir` is not created. I would expect it to be created if it doesn't exist already.